downloadservice/__init__.py

```diff
@@ -0,0 +1,2 @@
+from importlib import metadata
+__version__ = metadata.version("downloadservice")
```

Comment:
can you find-out the package name programmatically?

Reply:
I didn't know it was possible but I found this solution. I tested it with a few local tests and it seems to work fine.